components/inputs/input-text.js

```diff
@@ -605,12 +605,14 @@ class InputText extends LabelledMixin(FormElementMixin(SkeletonMixin(RtlMixin(Li
 			|| (this.unit && this.dir !== 'rtl');
 
 		if (firstSlotHasNodes) {
-			requestAnimationFrame(() => this._firstSlotWidth = firstContainer.getBoundingClientRect().width);
+			// Firefox can have trouble rendering the cursor if the padding is a decimal
+			requestAnimationFrame(() => this._firstSlotWidth = Math.ceil(firstContainer.getBoundingClientRect().width));
```

Comment:
Any chance this could be happening across other input components?

Reply:
So I _think_ all the input components use this component as their base and will get the fix as well if they were having issues.  The visual diffs only updated for `input-percent` and `input-text`, but I believe that's because the other inputs either don't use the `slot` or the padding didn't change for those tests.  But I'll confirm we don't have any inputs with their own custom slots not using `input-text`

Reply:
Yeah looks like we're good.  Everything either uses `input-text` or something else that uses `input-text`, except `textarea` and `input-time` which don't offer the slotted sides.  `input-date` and `input-time` also make sure they themselves are an integer width using `Math.ceil` (https://github.com/BrightspaceUI/core/blob/main/components/inputs/input-date.js#L196), so maybe they ran into similar issues at one point.